test/detect/angelscript/default.txt

```diff
@@ -23,13 +23,15 @@ namespace MyApplication
             m_arr.insertLast(1.0f);
             m_arr.insertLast(1.75f);
             m_arr.insertLast(3.14159f);
-            uint x = 0x7fff0000;
+            uint x = 0x7fff0000 + 2;
```

Comment:
`+ 2` ?

Reply:
Are you only adding code now to get the auto-detect stuff back in balance?

Reply:
Ah, sorry, this was a leftover from some testing I was doing. I'll remove it from the PR.